monkey/monkey_island/cc/server_utils/consts.py

```diff
@@ -36,7 +36,8 @@ def _get_monkey_island_abs_path() -> str:
 )
 MONGO_CONNECTION_TIMEOUT = 15
 
-DEFAULT_SERVER_CONFIG_PATH = str(Path(MONKEY_ISLAND_ABS_PATH, "cc", SERVER_CONFIG_FILENAME))
+PACKAGE_CONFIG_PATH = Path(MONKEY_ISLAND_ABS_PATH, "cc", SERVER_CONFIG_FILENAME)
+USER_CONFIG_PATH = Path(os.getcwd(), SERVER_CONFIG_FILENAME)
```

Comment:
I think we're misusing this `consts.py` file. These constants are only used in `cc/setup/config_setup.py`. There's no reason for them to be visible outside of that file; nothing else uses them. We should move our constants to the places where they're used if they're only used in one place, while this file can contain constants that are used by multiple modules.

We can discuss as a team and update our developer's guide with whatever conclusion we come to.